README.md

```diff
@@ -677,15 +677,68 @@ January 1, 2016
 
 #### Notes
 
+California approved a $2 cigarette tax increase. It takes effect April 1, 2017.
 Local taxes are not included and can be substantial.
 
 #### Source
 
 RJ Reynolds; state revenue departments; Tax Foundation
 
-### <a id="table25"></a>Table 25. State Spirits Excise Tax Rates (Dollars Per Gallon)
+### <a id="table26"></a>Table 26. State Vapor Excise Tax Rate
 
-January 1, 2016
+January 1, 2017
+
+This summary data is impractical for data analysis and has been excluded from the set of data. The summary table is reproduced below:
+
+| State | Tax Rate |
+| --- | --- | --- |
```

Comment:
This is part of [GitHub Flavored Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables)'s table syntax. It allows for the writing of tables in a pure text format. The second row here signified the separation between header and body in the table. In this case, the separation row includes one more cell than is in either the header or footer (3 instead of 2).

Change to:
```
| State | Tax Rate |
| --- | --- |
| Calif. | 27.30% of wholesale |
...etc...
```